lib/internal/Magento/Framework/Filesystem/Driver/File.php

```diff
@@ -962,7 +960,7 @@ public function getAbsolutePath($basePath, $path, $scheme = null)
         // check if the path given is already an absolute path containing the
         // basepath. so if the basepath starts at position 0 in the path, we
         // must not concatinate them again because path is already absolute.
-        if (0 === strpos($path, $basePath)) {
+        if ($path && strpos($path, $basePath) === 0) {
```

Comment:
Would it be better to add
```suggestion
        if ($path === null) {
            $path = '';
        }
        if (strpos($path, $basePath) === 0) {
```
instead of adding workarounds to the two methods?

Reply:
Hello @fascinosum 
Thank you for your review and proposition
However, we will have redundant operation `strpos('', $basePath)` in this case IMHO
What do you think about it?

Reply:
fixed as discussed, please check